fsh-generated/resources/StructureDefinition-au-erequesting-servicerequest-path.json

```diff
@@ -83,6 +83,73 @@
       },
       {
         "id": "ServiceRequest.bodySite",
+        "extension": [
```

Comment:
why is the obligation changing in this PR?

Reply:
This is an autogenerated file. No obligations were changed.

Reply:
how strange.